plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java

```diff
@@ -94,6 +95,11 @@ public DeltaLakeTableProperties(DeltaLakeConfig config)
                         "Enables deletion vectors",
                         config.isDeletionVectorsEnabled(),
                         false))
+                .add(stringProperty(
```

Comment:
We should use `ARRAY(VARCHAR)` type in my opinion.

Reply:
Let me update it